config/traject/pdc_describe_indexing_config.rb

```diff
@@ -16,6 +16,13 @@
   provide 'logger', Logger.new($stderr, level: Logger::WARN)
 end
 
+# Converting the XML to JSON is a bit expensive therefore we make that conversion
+# only once per record and save it to the context so that we can re-use it.
+each_record do |record, context|
+  xml = record.xpath("/hash").first.to_xml
+  context.clipboard[:record_json] = Hash.from_xml(xml).to_json
```

Comment:
Here is where we create the JSON string with the PDC data (we are taking it from the XML just as the code did before) but this prevents from having the data twice in the XML (one as XML elements and one as giant CDATA with a string that represents the JSON)